tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "module": "es2020",
+    "lib": ["dom", "es2020"],
+    "target": "es2020",
+    "moduleResolution": "node",
+    "strict": true,
+    "esModuleInterop": true,
+    "isolatedModules": true
+  },
+  "paths": {
+    "@maxgraph/core": ["packages/core/src/index.ts"],
+    "@maxgraph/ts-example": ["packages/ts-example/src/main.ts"]
```

Comment:
❓ are you sure we need this configuration? according to the documentation, if we keep it, we must also configure `baseUrl`: https://www.typescriptlang.org/tsconfig#paths

Reply:
Yes, I think you right! Added it to base config and removed from `core` package in d53ea1a